src/main/java/org/jenkinci/plugins/mock_slave/MockCloud.java

```diff
@@ -109,12 +126,20 @@ synchronized long newNodeNumber() {
             return "Mock Cloud";
         }
 
+        public FormValidation doCheckOneShot(@QueryParameter int numExecutors, @QueryParameter boolean oneShot) {
```

Comment:
The plugin does not provide an API, I am not bothering with that. It is not even intended for use in production systems anyway.